cl/tests/utils.py

```diff
@@ -57,3 +60,70 @@ def __init__(
         if mock_raw is True:
             file_stream = ContentFile("OK")
             self.raw = file_stream
+
+
+class MockACMSDocketReport:
+    def __init__(self, court_id):
+        pass
+
+    def _parse_text(self, json):
+        acms_data = DocketDataFactory(
+            court_id="ca2",
+            appeal_from="S.D.N.Y . (NEW YORK CITY)",
+            case_name="United States of America v. Raji",
+            case_type_information="Criminal, Direct Criminal",
+            date_filed=date(2023, 10, 2),
+            docket_number="23-6364",
+            fee_status="IFP Granted",
+            originating_court_information={
+                "identifier": "S.D.N.Y. (NEW YORK CITY)",
+                "name": "S.D.N.Y . (NEW YORK CITY)",
```

Comment:
All things being equal, we should not pick test data with this goofy spacing anomaly (`S.D.N.Y .` versus `S.D.N.Y.`).

Maybe it shouldn't be in the juriscraper tests at all, idk! But there exist other data that do not have this defect.